h/models/group_scope.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+
+from h._compat import urlparse
```

Comment:
What are you going to use `urlparse` for?

Reply:
No idea!